### PR TITLE
Fix process.env resolution

### PR DESCRIPTION
**Pull request by medikoo**
When working on Tencent commands, we've approached some issues, and after investigation it appeared that it's caused by fact of environment variables not being resolved from `.env` files

Ideally `.env` files should be resolved at earliest stage of a program, and further logic should simply refer to `process.env` for environment variables values.

This PR ensures it's the case.

As there's no test suite, I've tested manually by running few component commands. All worked without issues.

### CODE REVIEW

File: src/cli/commands/utils.js [hunk 1]
```diff
@@ -115,48 +99,13 @@ const loadInstanceCredentials = (stage) => {
...
-    isYaml = true
-  }
-  if (fileExistsSync(yamlFilePath)) {
-    filePath = yamlFilePath
-    isYaml = true
-  }
-  if (fileExistsSync(jsonFilePath)) {
-    filePath = jsonFilePath
-  }
+const loadAwsInstanceConfig = async (directoryPath) => {
```
Comment (ac360) [inline src/cli/commands/utils.js:102]: Components is multi-vendor and we don't want to create vendor-specific methods like this.

Comment (medikoo) [reply src/cli/commands/utils.js:102]: This is in AWS specific directory (note we separated commands into vendor specific folders)

Comment (ac360) [reply src/cli/commands/utils.js:102]: There is no AWS-specific directory is my point.  There can be Tencent-specific stuff, but everything else must be vendor agnostic.

Comment (medikoo) [reply src/cli/commands/utils.js:102]: Altough indeed I see there's also resolution for Google env vars and Tencent. Can it work on Google?

And I take for Tencent it's dead logic

Comment (medikoo) [reply src/cli/commands/utils.js:102]: Ok so I'll rename it to `loadVendorInstance Config`

Comment (medikoo) [reply src/cli/commands/utils.js]: Updated.

Note this is internal name, not exposed outside of module. It's just to not colide with more generic `loadInstanceConfig` which doesn't attempt to resolve vendor specific info.

**Pull request merged**